cparser/processor.js

```diff
@@ -81,7 +81,7 @@ module.exports = that = {
 			var missingFuncs = regexParser.getMissingDeclarations(cleanText);
 
 			var prototypesStr = missingFuncs.join('\n') + '\n'
-				+ '#line ' + linesBeforeInjection + '\n';
+				+ '#line ' + linesBeforeInjection + ' "' + inputFile + '"\n';
```

Comment:
Alright, let's go with absolute paths. It feels wrong to embed absolute paths inside the cpp but I can't quickly find a workaround.